stream-chat-android-client/src/test/java/io/getstream/chat/android/client/api/TokenAuthInterceptorTests.kt

```diff
@@ -1,6 +1,7 @@
 package io.getstream.chat.android.client.api
 
 import io.getstream.chat.android.client.api.FakeResponse.Body
+import io.getstream.chat.android.client.api.interceptor.TokenAuthInterceptor
```

Comment:
Here, the test package should be updated instead

Reply:
Nice catch